dm/dumpling/dumpling_test.go

```diff
@@ -146,21 +148,25 @@ func (t *testDumplingSuite) TestCallStatus(c *C) {
 	dumpConf.PromRegistry = tidbpromutil.NewDefaultRegistry()
 
 	s := m.Status(nil).(*pb.DumpStatus)
-	c.Assert(s.CompletedTables, Equals, float64(0))
-	c.Assert(s.FinishedBytes, Equals, float64(0))
-	c.Assert(s.FinishedRows, Equals, float64(0))
-	c.Assert(s.EstimateTotalRows, Equals, float64(0))
+	assert.Equal(t, s.CompletedTables, float64(0))
+	assert.Equal(t, s.FinishedBytes, float64(0))
+	assert.Equal(t, s.FinishedRows, float64(0))
+	assert.Equal(t, s.EstimateTotalRows, float64(0))
+	assert.Equal(t, s.Progress, "")
+	assert.Equal(t, s.Bps, int64(0))
```

Comment:
Any other test for non-zero value?

Reply:
it is inconvenient here and I added an integration for non-zero value.